combustion-android-ble/src/main/java/inc/combustion/framework/ble/uart/meatnet/NodeProbeStatusRequest.kt

```diff
@@ -67,7 +62,7 @@ internal class NodeProbeStatusRequest(
             val serialNumber: UInt = data.getLittleEndianUInt32At(SERIAL_NUMBER_INDEX)
             val hopCount: HopCount = HopCount.fromUByte(data[HOP_COUNT_INDEX])
 
-            return NodeProbeStatusRequest(requestId, payloadLength, serialNumber, hopCount, probeStatus)
+            return NodeProbeStatusRequest(requestId, payloadLength, Integer.toHexString(serialNumber.toInt()).uppercase(), hopCount, probeStatus)
```

Comment:
There're a few of these conversions around--does it make sense to `typealias` the serial number and create extension methods for it? No's an ok answer.